### PR TITLE
Adding additional instructions to the Docker Engine installation/inde…

### PR DESCRIPTION
**Pull request by mikebrow**
Docs update per discussions with @moxiegirl regarding #15823

Signed-off-by: Mike Brown brownwm@us.ibm.com

### CODE REVIEW

File: docs/installation/index.md [hunk 1]
```diff
@@ -1,33 +1,49 @@
...
-* [Install on Joyent Public Cloud](joyent)
-* [Mac OS X](mac)
 * [Oracle Linux](oracle)
-* [Rackspace Cloud](rackspace)
 * [Red Hat Enterprise Linux](rhel)
-* [IBM SoftLayer](softlayer)
 * [openSUSE and SUSE Linux Enterprise](SUSE)
 * [Ubuntu](ubuntulinux)
+
+## Other Linux
```
Comment (moxiegirl) [inline docs/installation/index.md]: @mikebrow Thanks for doing this.  Looks great.  I'd remove this heading `## Other Linux`

File: docs/installation/index.md [hunk 2]
```diff
@@ -1,33 +1,49 @@
...
+
+## On OSX and Windows
+* [Mac OS X](mac)
 * [Windows](windows)
 
+## The Docker Archives 
+Instructions for installing prior releases of Docker can be found in the following docker archives:
+[Docker v1.7](http://docs.docker.com/v1.7/), [Docker v1.6](http://docs.docker.com/v1.6/), [Docker v1.5](http://docs.docker.com/v1.5/), and [Docker v1.4](http://docs.docker.com/v1.4/).
+
+## Where to go next
```
Comment (moxiegirl) [inline docs/installation/index.md]: @mikebrow  We don't really need this section. If they haven't installed, it doesn't make sense to tell them to go somewhere next.

**Comment by moxiegirl** [comment]
ping @thaJeztah --- nice new org for this page

**Comment by SvenDowideit** [comment]
LGTM!

File: docs/installation/index.md (hunk 2 above)
Comment (mikebrow) [inline docs/installation/index.md]: Doesn't hurt either :-)  The other products machine, etc. have a where to go next section. It's just an emphasis to get them to get some training, read more docs after install etc.

File: docs/installation/index.md (hunk 1 above)
Comment (mikebrow) [inline docs/installation/index.md]: The binaries are sort of different than all the others... off the reservation sort of thing.  How best to separate it from the other linux installs?

Comment (moxiegirl) [inline docs/installation/index.md]: The new paragraph is enough.  You have a list of what we have...and this is   stated as an alternative to the list. That is why the heading is not needed.

File: docs/installation/index.md (hunk 2 above)
Comment (moxiegirl) [inline docs/installation/index.md]: It logically doesn't make sense. They haven't installed the product. The where to go next *is* the installation.  Some of our other pages have that not all of them. They are inconsistently used and so we will be coming up with some guidelines.

File: docs/installation/index.md (hunk 1 above)
Comment (thaJeztah) [inline docs/installation/index.md]: Agreed, I think the heading is redundant

File: docs/installation/index.md (hunk 2 above)
Comment (thaJeztah) [inline docs/installation/index.md]: Agreed that the "next" step would be the actual installation, but won't make a fuss if the links are kept :smile:

**Comment by thaJeztah** [comment]
LGTM after the "other linux" heading is removed, no *strong* opinion for the removal of the "where to go next" links.

@moxiegirl (slightly side-tracking here) I recently commented on another issue with some thoughts on improving the install docs; https://github.com/docker/docker/issues/15765#issuecomment-134408029.

File: docs/installation/index.md (hunk 2 above)
Comment (mikebrow) [inline docs/installation/index.md]: Right ... and the best place for those steps would be at the end of each of the install pages.. I was using the call stack approach where most folks come back to the main page after doing the sub page for downloading..  How about something like... after installing on your selected platform be sure to check out these pages for next steps:

File: docs/installation/index.md (hunk 1 above)
Comment (mikebrow) [inline docs/installation/index.md]: ok :-) will do

**Comment by mikebrow** [comment]
@moxiegirl comments have been addressed for the changes to the index.md file.  I've removed the "Other Linux" heading and Changed the "Next Steps" heading to "Where to go After Installing"

**Comment by moxiegirl** [comment]
@mikebrow Thanks for your patience. LGTM

**Pull request merged**